webapi/tct-extra-html5-tests/w3c/system-state-and-capabilities/the-navigator-object/get-navigatorlanguage-manual.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>NavigatorLanguage: navigator.language returns the user's preferred language</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/webappapis.html#navigatorlanguage">
+<script src="../../../resources/testharness.js"></script>
+<script src="../../../resources/testharnessreport.js"></script>
+<h2>Precondition</h2>
+<p>The user agent's preferred language is set as English (United Kingdom) (en-GB).</p>
```

Comment:
If you have to change, please refer to https://crosswalk-project.org/jira/browse/XWALK-2132 (putting this link into commit message too):

> Set Android phone language to Espanol (Estados Unidos)
> *Expected result: window.navigator.language is es-US